cypress/integration/notyf_spec.js

```diff
@@ -370,6 +370,20 @@ context('Notyf', () => {
   });
 
   describe('Public API', () => {
+
+    it('should dismiss one notification', () => {
```

Comment:
I'd also be nice to have a unit test for dismissing timeout notifications. Like, what happens when you fire 2 notifications, the first one times out and vanishes, but you still dismiss it via API? we should check the other notification shouldn't disappear. 

It could go something like this (pseudocode)

```
1. Fire two success notifications. One with duration: 500, the other with duration: 1000;
2. After 500ms, dismiss notification with index 0. 
3. Expect the number of notifications to still be equal to 1, because #2 should remain.

The trick here is to see that those timeout vanishes do not .splice the 'notyfNotifications' array so you still hold the reference to the non-existent notification
```